include/inviwo/core/io/serialization/deserializer.h

```diff
@@ -212,23 +234,66 @@ class IVW_CORE_API Deserializer : public SerializeBase, public LogFilter {
                      std::string_view itemKey = "item",
                      std::string_view comparisonAttribute = SerializeConstants::KeyAttribute);
 
-    template <typename K, typename V, typename C, typename A>
-    void deserialize(std::string_view key, std::map<K, V*, C, A>& sMap,
-                     std::string_view itemKey = "item",
-                     std::string_view comparisonAttribute = SerializeConstants::KeyAttribute);
-
-    template <typename K, typename V, typename C, typename A>
-    void deserialize(std::string_view key, std::map<K, std::unique_ptr<V>, C, A>& sMap,
-                     std::string_view itemKey = "item",
-                     std::string_view comparisonAttribute = SerializeConstants::KeyAttribute);
-
     template <typename K, typename V, typename H, typename C, typename A>
     void deserialize(std::string_view key, std::unordered_map<K, V, H, C, A>& map,
                      std::string_view itemKey = "item",
                      std::string_view comparisonAttribute = SerializeConstants::KeyAttribute);
 
-    template <typename T, typename ValueGetter, typename IdGetter>
-    void deserialize(std::string_view key, ContainerWrapper<T, ValueGetter, IdGetter>& container);
+    /**
+     * For more advanced deserialization. useful when one has to call observer
+     * notifications for example.
+     * Example usage, serialize as usual
+     * ```{.cpp}
+     *     s.serialize("TFPrimitives", values_, "point");
+     *
+     * ```
```

Comment:
```suggestion
    /**
     * For more advanced deserialization. Useful when one has to call observer notifications.
     * Example usage, serialize as usual
     * ```{.cpp}
     *     s.serialize("TFPrimitives", values_, "point");     *
     * ```
```